src/main/java/io/federecio/dropwizard/swagger/SwaggerBundleConfiguration.java

```diff
@@ -15,6 +15,7 @@
  */
 package io.federecio.dropwizard.swagger;
 
+import org.eclipse.jetty.util.URIUtil;
```

Comment:
Is it possible to use something like UriBuilder instead of this jetty internal class?